site/en/guides/border-router/espressif-esp32.md

```diff
@@ -1,69 +1,73 @@
 # ESP Thread Border Router
 
-Contributor: https://github.com/gjc13
+Contributor: https://github.com/gjc13 https://github.com/zwx1995esp
 
 ESP Thread border router is a FreeRTOS-based implementation running on a combination of Espressif's Wi-Fi and 802.15.4 SoCs.
 
 Hardware requirements:
 
-* An ESP32 series Wi-Fi dev kit. (ESP32, ESP32-C3, ESP32-S2 or ESP32-S3)
-* An [ESP32-H2](https://www.espressif.com/en/news/ESP32_H2) dev kit for running RCP.
+Espressif provides an [ESP Border Router Board](https://www.espressif.com/en/news/Thread_Border_Router_Certification) which integrates the host SoC and the RCP into one module.
 
-## Hardware connection
-
-The ESP32-H2 RCP will be connected to the border router via UART.
+## Hardware platforms
 
 <figure>
-<a href="../../guides/images/otbr-esp-connection.jpg"><img src="../../guides/images/otbr-esp-connection.jpg" width="600" border="0" alt="ESP32-H2 UART" /></a><figcaption>ESP32-H2 UART connection</figcaption>
+<a href="../../guides/images/otbr-esp-br-board.png"><img src="../../guides/images/otbr-esp-br-board.png" width="600" border="0" alt="ESP Thread Border Router Board" /></a><figcaption>ESP Thread Border Router Board</figcaption>
```

Comment:
Can you compress the new image a little bit.  It needs to be smaller than 1048576 bytes for importing into openthread.io

Reply:
Now the image is ~55000 bytes